third_party/move/move-compiler-v2/tests/checking/typing/entry_inline_err.move

```diff
@@ -0,0 +1,3 @@
+module 0x123::a{
+    public(friend) entry inline fun a(){}
```

Comment:
1. Could we *add* a test case where the function is used, i.e., there is a call to `a()`, so that we cover both the original situation (inline entry function defined but not called) and the new situation (inline entry function defined and called)? 
2. If we decide to only gatekeep this behavior for Move 2.0 and above, can we have a test check the behavior for Move 1.0 (e.g., by adding an appropriate `TestConfig` in `testsuite.rs`) in addition to Move 2.0 (current test behavior)? We could create a test folder for such tests where gatekeeping is done based on language versions.

Reply:
done